djxml/xmlmodels/base.py

```diff
@@ -11,11 +11,10 @@
 from django.core.exceptions import (ObjectDoesNotExist, FieldError,
                                     MultipleObjectsReturned,)
 from django.db.models.base import subclass_exception
-try:
-    from django.utils.encoding import (
-        smart_bytes as smart_str, force_text as force_unicode)
-except ImportError:
-    from django.utils.encoding import smart_str, force_unicode
+
+from django.utils.encoding import force_text
+from django.utils.encoding import smart_bytes, smart_text
+smart_str = smart_text if six.PY3 else smart_bytes
```

Comment:
Minor point and non-blocking, but I think this is a PEP8 violation; this should come after the `from .loading import` below and be preceded by two new-lines.